Gemfile.lock

```diff
@@ -405,69 +408,70 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
+    jsonpath (1.1.5)
```

Comment:
### [jsonpath](https://github.com/joshbuddy/jsonpath)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.2 → 1.1.5

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/jsonpath/1.1.2/1.1.5)_]





<details>
<summary>Commits</summary>

A change of **30** commits. See the full changes on [the compare page](https://github.com/joshbuddy/jsonpath/compare/v1.1.2...v1.1.5).

These are the first 10 commits:
- (c7d7f6b) [skip when node size is zero](https://github.com/joshbuddy/jsonpath/commit/c7d7f6b28bfcfad7618f19754795e7d9fb2b85f1)
- (b526cdc) [fix hanging on LF character](https://github.com/joshbuddy/jsonpath/commit/b526cdc5279e6afdf32863d097b8f1e8d2d808ad)
- (268c672) [Jsonpath with keyname $ not working](https://github.com/joshbuddy/jsonpath/commit/268c67201998c106222de778f2f6c7f887e76df5)
- (9eb3d04) [Merge pull request #157 from anuravi98/master](https://github.com/joshbuddy/jsonpath/commit/9eb3d04d72ef485542c99140d1f8bc934fd82468)
- (6a3f044) [Merge pull request #155 from ipost/master](https://github.com/joshbuddy/jsonpath/commit/6a3f044b65aed8e209a03927c9bcdf5fce4c9f3e)
- (9068f9b) [Merge pull request #154 from a5-stable/aviod-zero-division-e…](https://github.com/joshbuddy/jsonpath/commit/9068f9b61c83e541cd997db4a7d82c355d1d74c7)
- (2c7cea8) [Fix method typo](https://github.com/joshbuddy/jsonpath/commit/2c7cea84fb076eefd16314166adaafc74f8efd95)
- (804a5af) [Handle any failure to match a token](https://github.com/joshbuddy/jsonpath/commit/804a5afa1b3e13cbc11ce4983fc61927b4e7da97)
- (4e31ac9) [Bump version](https://github.com/joshbuddy/jsonpath/commit/4e31ac9fc05eb309999903e674cd3a79763cb7a0)
- (bae8ab0) [Update enumerable.rb](https://github.com/joshbuddy/jsonpath/commit/bae8ab08b528362d45f59584f9fd42f022cdddf0)


</details>